src/test/scala/uk/gov/nationalarchives/DynamoFormattersTest.scala

```diff
@@ -21,6 +22,15 @@ class DynamoFormattersTest extends AnyFlatSpec with TableDrivenPropertyChecks wi
       parentPath -> AttributeValue.fromS("testParentPath"),
       name -> AttributeValue.fromS("testName"),
       typeField -> AttributeValue.fromS("ArchiveFolder"),
+      transferringBody -> AttributeValue.fromS("testTransferringBody"),
```

Comment:
Again this is personal preference so feel free to ignore me but if you import `import software.amazon.awssdk.services.dynamodb.model.AttributeValue._` then you can just reference `fromS`, `fromN` and `fromL` which might make it easier to read.